components/webext-storage/src/sync/mod.rs

```diff
@@ -24,19 +23,17 @@ pub fn is_default<T: PartialEq + Default>(v: &T) -> bool {
     *v == T::default()
 }
 
-// XXX - need to work out how to consolidate "payload" vs "bso", particularly
-// the timestamp, etc. For now, we represent this in ServerPayload, even though
-// that's never actually stored in this way.
-// XXX - this is going to be replaced with envelopes!
 #[derive(Debug, Serialize, Deserialize)]
-pub struct ServerPayload {
+// TODO: Should we camelize record properties? It's what we do for other
+// records, but, since this is the first engine where we'll use Rust
+// everywhere, it doesn't really matter.
+// #[serde(rename_all = "camelCase")]
```

Comment:
I'm kind of tempted to say "no, don't camelize" but calling it out just in case.

Reply:
Yeah, 🤷 from me. I guess I still lean towards that we should because even though we've kinda eliminated 'lastModified' and 'sortIndex' from here, they aren't going to go away so it's always going to be odd when we don't camelize.

(In this case, it's literally just 'ext_id' -> 'extId', right? Maybe we should just rename it to 'ext' ;)

Reply:
OK, I went ahead and added `rename_all = "camelCase"`. 😊 The others are spelled `modified` and `sortindex` (😆), so could go either way! We could also rename to `extid` to match `sortindex`, but I don't want to do that! 😂